docs/docs/how-to-deploy.mdx

```diff
@@ -136,6 +138,87 @@ The `<custom_image_tag>` should reference how this image will be different from
+   - `DOCKER_HUB_LOGIN` - a login name for DockerHub
+   - `DOCKER_HUB_PASSWORD` - a password for DockerHub
+
+2. In your GitHub repository create a file [`.github/workflows/action_server.yml`](https://github.com/RasaHQ/action-server-gha/blob/master/examples/action_server.yml).
+
+The GitHub Action workflow below builds a new docker image every time when new changes are pushed into the `master` branch.
+
+```yaml
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    name: Build Action Server image and upgrade Rasa X deployment
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - id: action_server
+      name: Build an action server with a custom actions
+      uses: RasaHQ/action-server-gha@master
+      # Full list of parameters: https://github.com/RasaHQ/action-server-gha/tree/master#input-arguments
+      with:
+        docker_image_name: 'account_username/repository_name'
+        docker_registry_login: ${{ secrets.DOCKER_HUB_LOGIN }}
+        docker_registry_password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+        # More details about github context:
+        # https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context
+        #
+        # github.sha - The commit SHA that triggered the workflow run
+        docker_image_tag: ${{ github.sha }}
+```
+
+3. Push your changes to the `master` branch. After changes are pushed, the workflow will build and push a new image into the DockerHub registry.
+
+4. Now, you can use your new brand docker image or, for example, extend your workflow.
+The example below shows how to extend your workflow by a new step that updates a Rasa X deployment based on [Helm Chart](https://rasa.com/docs/rasa-x/installation-and-setup/customize/#adding-a-custom-action-server)
+
+```yaml
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    name: Build Action Server image and upgrade Rasa X deployment
+    steps:
+    [..]
+
+    # This step shows only the example of output parameter usage
+    # and it's not focused on deployment itself.
+    - name: "Upgrade a Rasa X deployment"
+      run: |
+        helm upgrade --install --reuse-values \
+          --set app.name=${{ steps.action_server.outputs.docker_image_name }} \
+          --set app.tag=${{ steps.action_server.outputs.docker_image_tag }} rasa rasa-x/rasa-x
+```
+
+As you can see it's possible to use output variables from the `action_server` step. The `steps.action_server.outputs.docker_image_name` variable returns
+a docker image name and the `steps.action_server.outputs.docker_image_tag` variable returns a docker image tag.
+
+More examples on how to use and customize [Rasa Action Server GitHub Action](https://github.com/RasaHQ/action-server-gha) you can find in the [Rasa Action Server GitHub Action](https://github.com/RasaHQ/action-server-gha) repository.
```

Comment:
Action server sounds really similar to GitHub Actions. It can be confusing. What if we called this repo the `Rasa GitHub Actions` repository. This opens up the possibility of different actions in the future as well

Reply:
I agree, too many actions in one name 😄